config/crd/bases/secops.kubewarden.io_clusteradmissionpolicies.yaml

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
The name of this file is in purpose?

Reply:
Sorry, that's a leftover (I initially though about calling everything like `secops.kubewarden.io` instead of `policies.kubewarden.io`).

I've removed it from the PR

Reply:
Thanks!